### PR TITLE
adding sonar properties

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

Setting up sonar properties for sonar qube reporting

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/ecrc-frontend/sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+sonar.exclusions=**/*.test.js, **/AuthenticationGuard.js, **/*.test.jsx, node_modules/**, coverage/**, bower_components/**, jspm_packages/**, typings/**, lib-cov/**
+
+# Test configurations
+sonar.tests=src/components
+sonar.test.inclusions=**/*.test.js, **/*.test.jsx
+
+# Reports
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+sonar.testExecutionReportPaths=coverage/test-report.xml
+# sonar.coverage.exclusions=
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/sonar-project.properties:26]: nit: missing newline

Comment (akroon3r) [reply src/ecrc-frontend/sonar-project.properties]: resolved

**Comment by sdevalapurkar-bcgov** [comment]
failing jest tests.... something about jest-sonar-reporter but things look good on my end, not sure what's wrong

**Comment by akroon3r** [comment]
> failing jest tests.... something about jest-sonar-reporter but things look good on my end, not sure what's wrong

missing dev dependency in package.json

**Pull request merged**